functions/__sf_section_jobs.fish

```diff
@@ -0,0 +1,44 @@
+# Jobs
+#
+
+function __sf_section_jobs -d "Show icon, if there's a working jobs in the background."
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_JOBS_SHOW true
+    __sf_util_set_default SPACEFISH_JOBS_PREFIX ""
+    __sf_util_set_default SPACEFISH_JOBS_SUFFIX " "
+    __sf_util_set_default SPACEFISH_JOBS_SYMBOL ✦
+    __sf_util_set_default SPACEFISH_JOBS_COLOR "blue"
```

Comment:
To keep consistent with other strings, we should remove the quotes around `blue` 😄